lib/TPP/DefaultTppPasses.cpp

```diff
@@ -416,6 +417,12 @@ struct DefaultTppPasses : public DefaultTppPassesBase<DefaultTppPasses>,
       pm.addNestedPass<func::FuncOp>(createTppConversionPass());
       pm.addNestedPass<func::FuncOp>(createCleanupPass());
 
+      // Decompose Aggregated operations. These ops currently do not
+      // bufferize. Once this is possible we can move this pass after
+      // bufferization.
+      pm.addNestedPass<func::FuncOp>(createDecomposeAggregatedOpsPass());
```

Comment:
Maybe this could be done before TPP conversion in case any of the individual ops match.
But not sure if we really care atm.

Reply:
moved. Let's see, once we have e2e tests we can reshuffle if needed.